src/azul/terraform.py

```diff
@@ -776,6 +784,14 @@ def tf_config(self, app_name):
             resource['source_code_hash'] = '${filebase64sha256("%s")}' % package_zip
             resource['filename'] = package_zip
 
+            # Inject a Lambda function alias resource pointing the alias at the
+            # latest published version of the function.
```

Comment:
```suggestion
            # Inject a function alias resource pointing the alias at the
            # latest published version of the function.
```

At this stage, it should be clear from the context what the term *function* refers to.